docker/deploy.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -euo pipefail
+
+image_name=$1
```

Comment:
Isn't this the image tag not image name?

Reply:
I copied the script from dex-services but I think you're right.